bciers/apps/reporting/src/data/jsonSchema/newEntrantInformation.tsx

```diff
@@ -0,0 +1,302 @@
+import { RJSFSchema, UiSchema, WidgetProps } from "@rjsf/utils";
+
+import { Typography } from "@mui/material";
+import {
+  ArrayFieldTemplate,
+  FieldTemplate,
+  TitleOnlyFieldTemplate,
+} from "@bciers/components/form/fields";
+import checkboxWidget from "@bciers/components/form/widgets/CheckboxWidget";
+import { DateWidget } from "@bciers/components/form/widgets";
+import { ReadOnlyWidget } from "@bciers/components/form/widgets/readOnly";
+import React from "react";
+import ObjectFieldTemplate from "@rjsf/core/lib/components/templates/ObjectFieldTemplate";
+
+interface FieldTemplateProps {
+  id: string;
+  classNames: string;
+  children: React.ReactNode;
+  formContext: any;
+}
+export const ProductionDataTitleWidget: React.FC<WidgetProps> = ({
+  id,
+  value,
+}) => {
+  return (
+    <div id={id} className="w-full mt-8">
+      <h2 className="inline-block p-0 text-lg font-bold text-bc-bg-blue m-0 mb-12">
+        <u>Product:</u> {value}
+      </h2>
+    </div>
+  );
+};
+/**
+ * Utility function to fetch the associated emission name dynamically
+ * based on the field ID and the form context.
+ * @param {string} fieldId - ID of the field
+ * @param {any} formContext - Context of the form, containing all data
+ * @returns {string | null} - Returns the emission name or null
+ */
+const getAssociatedEmissionName = (
```

Comment:
This relies on an implementation detail of RJSF and will likely not work the moment we update the library.
Happy to talk about how we could do what you're trying to do